keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlLifecycleMonitorRepository.kt

```diff
@@ -78,20 +78,23 @@ class SqlLifecycleMonitorRepository(
     }
 
   fun LifecycleEvent.getUid(jooq: DSLContext): String {
```

Comment:
This is an extension function, so you don't have to pass in `jooq`...

Reply:
Oh nice! :D